src/viewer.js

```diff
@@ -1294,7 +1294,8 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
                     alwaysBlend: _this.alwaysBlend,
                     minPixelRatio: _this.minPixelRatio,
                     crossOriginPolicy: _this.crossOriginPolicy,
-                    debugMode: _this.debugMode
+                    debugMode: _this.debugMode,
+                    placeholderFillStyle: _this.placeholderFillStyle
```

Comment:
You should also be able to include placeholderFillStyle as part of your image definition as passed into addTiledImage (and therefore also open() and the `tileSources` option on creation). That should override the global option for the viewer.